index.html

```diff
@@ -0,0 +1,168 @@
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+            <div class="card-work-item">
+                <div class="individual-card-item">
+                    <h3>Multi-Post Stories</br>Gain+Glory</h3>
+                    <ul class="languages">
+                        <li>Ruby on Rails</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                        <li>HTML</li>
+                    </ul>
+                    <button type="button" class="btn">See project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-me">
+            <h2>About Me</h2>
+            <p> I can help you build a product, feature or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded,
+                don't hestiate to contact me.</p>
+            <div class="resume-btn"><a href="#">Get My Resume</a></div>
+        </div>
+        <div class="about-me-skils">
+            <div class="worklanguages">
+                <img src="./images/icon-languages.svg" alt="Image depicting languages">
```

Comment:
- [ ] Well done on implementing the icons in your design! However, it seems that the icons are currently displayed as images with shadows. I would suggest using icons images only and applying the desired shadow effect through CSS code rather than relying on pre-generated images. This approach will give you more flexibility and control over the styling of your icons. This will help you achieve the desired shadow effect while maintaining a lightweight and easily customizable solution. Keep up the good work! 👍